### PR TITLE
Fix navigation issue

### PR DESCRIPTION
**Pull request by lucperkins**
This PR fixes an issue where documentation links are broken in the mobile view on some pages.

### CODE REVIEW

**Comment by yurishkuro** [comment]
can you elaborate on what problem this is solving?

**Comment by lucperkins** [comment]
@yurishkuro If you go to a page link https://jaegertracing.io/roadmap, the documentation links in the mobile view are broken.

**Comment by yurishkuro** [comment]
netlify build seems to be not reporting status (the actual build was successful https://app.netlify.com/sites/jaegertracing/deploys/5e597a5cac51d20008d22f5b)

**Comment by yurishkuro** [comment]
also netlify preview looks all css-broken https://5e597a5cac51d20008d22f5b--jaegertracing.netlify.com/

**Comment by lucperkins** [comment]
@yurishkuro I'm not sure what's up with that but the deploy preview for the PR is working fine:

https://deploy-preview-374--jaegertracing.netlify.com

File: themes/jaeger-docs/layouts/partials/navbar.html [hunk 1]
```diff
@@ -23,7 +24,7 @@
       </div>
     </div>
 
-    <div id="topNav" class="navbar-menu">
+    <div id="topNav" class="navbar-menu is-active">
```
Comment (yurishkuro) [inline themes/jaeger-docs/layouts/partials/navbar.html]: I think this one is flipping menu to active on resize

Comment (lucperkins) [reply themes/jaeger-docs/layouts/partials/navbar.html]: Whoops! Ugh, I keep doing this. Fixed.

**Pull request merged**